### DISCUSSION TITLE
prisma-fmt binary for neovim

### DISCUSSION THREAD

**Discussion by nikero41** [prisma/prisma#19360] [Q&A] [answered]
How can I download or add prisma-fmt to $PATH in order to use it inside NeoVim?

**Comment by nurul3101** [comment]
Hello @snikoletopoulos 👋 

Did you had a look at this section for NeoVim Editor setup: [Reference](https://www.prisma.io/docs/guides/development-environment/editor-setup#neovim)?

**Reply by benediktms** [reply]
I think many Neovim users no longer use CoC and favour Neoim's builtin LSP tooling. It wouldbe great to get access to a freestanding binary so that a formatter can be added to a Treesitter and native LSP based setup.

**Reply by nurul3101** [reply, marked as answer]
@benediktms Do you mind opening a GitHub issue for supporting freestanding binary here?
https://github.com/prisma/vim-prisma/issues

**Reply by nurul3101** [reply]
Thanks for creating the feature request! 🙌 
- https://github.com/prisma/vim-prisma/issues/13
